src/IO/AssimpLoader/AssimpCameraDataLoader.cpp

```diff
@@ -130,7 +129,7 @@ Core::Matrix4 AssimpCameraDataLoader::loadCameraFrame( const aiScene* scene,
     return frame;
 }
 
-void AssimpCameraDataLoader::fetchName( const aiCamera& camera, Camera& data ) const {
+void AssimpCameraDataLoader::fetchName( const aiCamera& camera, Camera& ) const {
```

Comment:
I'm not sure to understand what is the intent of this method.

Reply:
neither do I, it was done before this PR, here is just a "fix warning ...
todo in another PR ? I'll do a quick check if this name is meaningful and easy to propagate to component ...

Reply:
after a quick look, need a refactor of camera loading to feed camera manager component names.